doc/developer/interface/webpack/index.md

```diff
@@ -2,11 +2,53 @@
 layout: default
 ---
 
-
 Webpack
-==========================================================
+=======
+
+[Webpack](https://webpack.js.org) is used to bundle front end assets, including JavaScript, CSS, SCSS, images and fonts into static assets to be consumed.  It is also an integral part of the IRIDA internationalization system. A webpack plugin, `i18nThymeleafWebpackPlugin`, developed in house is used to capture strings that need to be internationalized for each webpack entry, combining them into an html Thymeleaf template that will get processed and translated at runtime.
+
+Due to the nature of webpack code splitting and file chunking to optimize file size, an indeterminate amount of resource files (css and js) can be created for each entry.  Once this process is complete, webpack creates a `manifest.json` file (portion shown below) that contains a json object for teach of the entries that shows all assets that are required for css, js, and html (internationalizations):
+
+```json
+{
+  "entrypoints": {
```

Comment:
This documentation should be updated to reflect the new format

Reply:
Updated in 25b9b61